lib/order.rb

```diff
@@ -0,0 +1,29 @@
+require "table_flipper"
+
+class Order 
+
+    attr_reader :id
+    attr_accessor :products, :customer, :fulfillment_status
+
+    #
+    def initialize(id, products, customer, fulfillment_status = :pending) 
+        good_values = [:pending, :paid, :processing, :shipped, :complete]
+        if !good_values.include?(fulfillment_status)
+            raise ArgumentError, "#{fulfillment_status} is not an acceptable value for the fulfillment parameter."
+        end
+        @id = id
+        @products = products
+        @customer = customer
+        @fulfillment_status = fulfillment_status
+    end
+    def total
+        total = ((@products.values.sum) * 1.075).round(2)
```

Comment:
This is good that you're making a local variable named `total` and assigning that value to that variable. However, in this method, we probably don't need that-- the important thing is that we get the value of this method to OUTSIDE of the method. The best way to do this is to do an explicit return. Therefore, I might prefer to see something like this:

```ruby
def total
  return ((@products.values.sum) * 1.075).round(2)
end
```

OR
```ruby
def total
  total = ((@products.values.sum) * 1.075).round(2)
  return total
end
```